apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -911,44 +911,54 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     |> Enum.uniq()
   end
 
-  @spec get_ifes_to_piggyback(ExitProcessor.Request.t(), __MODULE__.t()) :: list(Transaction.Signed.t())
+  @spec get_ifes_to_piggyback(ExitProcessor.Request.t(), __MODULE__.t()) ::
+          list({InFlightExitInfo.t(), Transaction.Signed.t()})
   defp get_ifes_to_piggyback(
          %ExitProcessor.Request{blocks_result: blocks},
          %__MODULE__{in_flight_exits: ifes}
        ) do
     known_txs = get_known_txs(blocks)
 
     ifes
-    |> Map.values()
-    |> Stream.map(fn %InFlightExitInfo{tx: signed_tx} -> signed_tx end)
+    |> Stream.filter(fn {_, %InFlightExitInfo{is_active: is_active}} -> is_active end)
```

Comment:
I think the flows can be cleaner here:
1/ keep `Map.values`, as you don't need the keys ever
2/ keep passing around `ife`, instead of the tuple `ife, signed_tx`
3/ pattern-match the `signed_tx` field out of `ife` only when needed, instead of pattern-matching it out of the tuple.

The tuple passed through the pipeline is redundant (ife contains signed tx)